src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/SalesforceOutputFormat.java

```diff
@@ -46,7 +44,9 @@ public RecordWriter<NullWritable, CSVRecord> getRecordWriter(TaskAttemptContext
     try {
       return new SalesforceRecordWriter(taskAttemptContext);
     } catch (AsyncApiException e) {
-      throw new RuntimeException("There was issue communicating with Salesforce", e);
+      throw new RuntimeException(
```

Comment:
Is the cause captured here correctly? Do we need to check for IApiFault ?

Reply:
IAPIFault will work for ConnectionException